server/src/migrations/1643049715370-constraints.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class constraints1643049715370 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE cards ALTER COLUMN section_id TYPE integer;
+      ALTER TABLE cards ALTER COLUMN section_id DROP DEFAULT;
+      ALTER TABLE cards ADD CONSTRAINT section_fk
+        FOREIGN KEY (section_id)
+          REFERENCES sections (id);
+      ALTER TABLE cards ALTER COLUMN title TYPE varchar(256);
```

Comment:
limit the length on the db title columns will allow for better DB performance in the long term. I also added FKs to the existing tables. This will give better data integrity.